models/tmp/stg_twilio__account_history_tmp.sql

```diff
@@ -1,2 +1,2 @@
-select * 
+select {{ dbt_utils.star(source('twilio', 'account_history')) }}
```

Comment:
This may be a bit drastic, but I would actually recommend we make this a patch upgrading in order to avoid unexpectedly breaking customers workflows if they are currently overriding the variables. See [this Mixpanel issue](https://github.com/fivetran/dbt_mixpanel/issues/44) where a customer actually saw unexpected failures as a result of a similar change.

Reply:
It would also be relevant to highlight this in particular within the CHANGELOG so users are immediately aware of this change.

Reply:
updated!